tools/next-version

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const { program } = require('commander')
+
+const VERSION_FILE = 'version.json'
+
+program
+  .option('-d, --debug', 'enable extreme logging')
+
+const calculateNewVersion = () => {
+  const versionFile = fs.readFileSync(VERSION_FILE)
+  // versionCode is just used in the integreat-react-native-app
+  const { versionName, versionCode } = JSON.parse(versionFile)
+  const versionNameParts = versionName.split('.').map(it => parseInt(it))
+
+  const date = new Date()
+  const year = date.getFullYear()
+  const month = (date.getMonth() + 1)
+
+  const versionNameCounter = year === versionNameParts[0] && month === versionNameParts[1] ? versionNameParts[2] + 1 : 0
+  const newVersionName = `${year}.${month}.${versionNameCounter}`
+  const newVersionCode = versionCode ? versionCode + 1 : undefined
```

Comment:
When/Why should this be undefined? Maybe also add a comment

Reply:
This is undefined for the webapp. The script is just copied from there.

Reply:
There is already a comment some lines above.